dev/WindowsAppSDK_Insights/WindowsAppSDKInsights.h

```diff
@@ -10,8 +10,50 @@
 #error "WIL Tracelogging.h must not be explicitly included when including this file"
 #endif
 
-char* WINDOWSAPPSDK_PACKAGE_VER();
-char* WINDOWSAPPSDK_EXPERIMENTATION_LEVEL();
+#include <wil/resource.h>
+#include <string>
+
+#define _WINDOWSAPPSDK_PACKAGE_VER_RESOURCEID             10000
+#define _WINDOWSAPPSDK_EXPERIMENTATION_FLAG_RESOURCEID    10001
+
+#define INSIGHTSRESOURCEDLL "Microsoft.WindowsAppSDK.Insights.Resource.dll"
+
+inline std::string load_resource_string(const char* dllName, uint32_t id)
+{
+    wil::unique_hmodule module(::LoadLibraryA(dllName));
+
+    if (module)
+    {
+        char buffer[256];
+        if (0 < LoadStringA(module.get(), id, buffer, ARRAYSIZE(buffer)))
+        {
+            return buffer;
+        }
+        else
+        {
+            LOG_LAST_ERROR_MSG("Unable to load resource string, id: %d, error: %d", id, GetLastError());
```

Comment:
>error: %d [](http://example.com/codeflow?start=72&length=9)

why log again?

Reply:
We need to know if any issues loading resources.

Reply:
LOG_LAST_ERROR_MSG is already logging last error